server/service/src/sync/translations/name.rs

```diff
@@ -397,20 +430,56 @@ impl SyncTranslation for NameTranslation {
             ));
         }
 
-        let patient_type = match r#type {
+        let properties = NameRowRepository::new(connection)
+            .find_one_oms_fields_by_id(&id)?
+            .and_then(|row| row.properties);
+
+        // Check if this record has a supply_level property
+        let has_supply_level = properties
+            .as_ref()
+            .and_then(|json_str| serde_json::from_str::<serde_json::Value>(&json_str).ok())
+            .and_then(|v| v.get("supply_level").cloned())
+            .is_some();
+
+        // Only push if it's a patient OR (is a store/customer AND has a supply_level property)
+        let is_patient = r#type == NameRowType::Patient;
+        let is_store = r#type == NameRowType::Store;
+        let is_customer_type = is_customer;
+
+        if !is_patient && !(is_store && has_supply_level) && !(is_customer_type && has_supply_level)
```

Comment:
Noticed that this was initially meant for patients only. I added logic to support cases for supply_levels since we would like to see that reflect in OG too. Either this solution or a dedicated translation for it

Reply:
I'm still not getting the category 5 shown in OG... 
<img width="448" height="140" alt="Screenshot 2025-11-04 at 8 34 37 AM" src="https://github.com/user-attachments/assets/ee3e6bb9-5410-406f-a712-fa90e1c1f054" />

<img width="656" height="524" alt="Screenshot 2025-11-04 at 8 35 12 AM" src="https://github.com/user-attachments/assets/eb3a87e0-cb3b-49f8-a2a7-a9f38798ccb3" />